docs/data/pages.ts

```diff
@@ -23,144 +23,171 @@ const pages: MuiPage[] = [
-          { pathname: '/x/react-data-grid/filtering/quick-filter' },
-          { pathname: '/x/react-data-grid/filtering/server-side', title: 'Server-side filter' },
-          { pathname: '/x/react-data-grid/filtering/multi-filters', plan: 'pro' },
-          {
-            pathname: '/x/react-data-grid/filtering/header-filters',
-            plan: 'pro',
-            newFeature: true,
-          },
-          { pathname: '/x/react-data-grid/filtering-recipes', title: 'Recipes' },
+          { pathname: '/x/react-data-grid/style', title: 'Styling basics' },
+          { pathname: '/x/react-data-grid/style-recipes', title: 'Styling recipes' },
+          { pathname: '/x/react-data-grid/layout' },
+          { pathname: '/x/react-data-grid/overlays', title: 'Overlays' },
+          { pathname: '/x/react-data-grid/components', title: 'Custom subcomponents' },
         ],
       },
-      { pathname: '/x/react-data-grid/pagination' },
       {
-        pathname: '/x/react-data-grid/selection',
+        pathname: '/x/react-data-grid/interactivity-group',
+        subheader: 'Interactivity',
         children: [
-          { pathname: '/x/react-data-grid/row-selection' },
-          { pathname: '/x/react-data-grid/cell-selection', plan: 'premium', newFeature: true },
+          { pathname: '/x/react-data-grid/localization' },
+          {
+            pathname: '/x/react-data-grid/custom-behavior',
+            children: [
+              { pathname: '/x/react-data-grid/api-object', title: 'API object' },
+              { pathname: '/x/react-data-grid/events' },
+              { pathname: '/x/react-data-grid/state' },
+            ],
+          },
         ],
       },
       { pathname: '/x/react-data-grid/export' },
       { pathname: '/x/react-data-grid/clipboard', title: 'Copy and paste', newFeature: true },
-      { pathname: '/x/react-data-grid/overlays', title: 'Overlays' },
-      { pathname: '/x/react-data-grid/components', title: 'Custom subcomponents' },
-      {
-        pathname: '/x/react-data-grid/style-group',
-        title: 'Style',
-        children: [
-          { pathname: '/x/react-data-grid/style', title: 'Overview' },
-          { pathname: '/x/react-data-grid/style-recipes', title: 'Recipes' },
-        ],
-      },
-      { pathname: '/x/react-data-grid/localization' },
       { pathname: '/x/react-data-grid/scrolling' },
-      { pathname: '/x/react-data-grid/virtualization' },
-      { pathname: '/x/react-data-grid/accessibility' },
-      { pathname: '/x/react-data-grid/performance' },
-      { pathname: '/x/react-data-grid/tree-data', plan: 'pro' },
-      { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
-      { pathname: '/x/react-data-grid/aggregation', plan: 'premium' },
-      { pathname: '/x/react-data-grid/pivoting', plan: 'premium', planned: true },
+
       {
         pathname: '/x/react-data-grid/list-view',
         title: 'List view',
         plan: 'pro',
         unstable: true,
       },
```

Comment:
This should be under **Advanced features**, I think.

Reply:
I think the same is true for Export, Clipboard and Scrolling pages